.jenkins/github/functional-sdk-tests.jenkinsfile

```diff
@@ -0,0 +1,113 @@
+@Library('c8y-common-steps') _
+
+def resolvedScm
+
+pipeline {
+  agent {
+    kubernetes {
+      yaml '''
+        apiVersion: v1
+        kind: Pod
+        spec:
+          imagePullSecrets:
+          - name: "ci-registry-pull"
+          containers:
+          - name: "java"
+            image: "registry.stage.c8y.io/ci/jdk:11"
+            command: ["/bin/sh"]
+            args: ["-c", "cat"]
+            tty: true
+            env:
+            - name: JENKINS_URL
+              value: http://jenkins.ci.svc.cluster.local:8080/
+            - name: MAVEN_USER_HOME
+              value: /home/jenkins/agent
+            - name: DOCKER_HOST
+              value: tcp://localhost:2375
+            workingDir: "/home/jenkins/agent"
+            resources:
+              requests:
+                cpu: 2
+                memory: 3Gi
+              limits:
+                cpu: 3
+                memory: 3Gi
+        '''
+    }
+  }
+
+  options {
+    timeout(time: 4, unit: 'HOURS')
```

Comment:
optional: here we can decrease this timeout. Currently tests are executing around 5 min.